python/ray/util/client/server/server.py

```diff
@@ -63,36 +62,24 @@ def Init(self, request, context=None) -> ray_client_pb2.InitResponse:
             job_config = pickle.loads(request.job_config)
             job_config.client_job = True
         else:
-            job_config = None
+            job_config = JobConfig()
```

Comment:
I put it None before just to keep all the code path rest run the same way as before. I'd prefer JobConfig as well, so if you think it's not an issue, just go to this one.